src/StatusQ/Components/private/statusMessage/StatusMessageHeader.qml

```diff
@@ -78,7 +87,7 @@ Item {
         }
         StatusBaseText {
             id: tertiaryDetailText
-            visible: !root.amISender
+            visible: !root.amISender && messageOriginInfo == ""
```

Comment:
> Because importing from other platforms requires those message to be
signed by a Status Account, it's undesirable to render the public key of
the account that signed imported messages.

> Hence, when there's a messageOriginInfo, we don't render the elided
public key.